console.go

```diff
@@ -2,26 +2,64 @@ package otto
 
 import (
 	"fmt"
-	"os"
-	"strings"
+	"io"
 )
 
-func formatForConsole(argumentList []Value) string {
-	output := []string{}
-	for _, argument := range argumentList {
-		output = append(output, fmt.Sprintf("%v", argument))
-	}
-	return strings.Join(output, " ")
+type Console interface {
```

Comment:
Lets describe the interface.
```suggestion
// Console is implemented by type which can be used to process console output.
type Console interface {
```